src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -41,7 +41,7 @@ internal partial class XamlCodeGeneration
 
 		// Determines if the source generator will skip the inclusion of UseControls in the
 		// visual tree. See https://github.com/unoplatform/uno/issues/61
-		private bool _skipUserControlsInVisualTree = true;
+		private bool _skipUserControlsInVisualTree = false;
```

Comment:
👍 👍 👍

Reply:
Arguably should be flagged as a breaking change though?

Reply:
It is, in the commit comments :)